### PR TITLE
Fixed the Drop-Down alignment and Button Design

### PR DESCRIPTION
**Pull request by sanjam-deriv**

### CODE REVIEW

File: css/main.scss [hunk 1]
```diff
@@ -1699,10 +1698,10 @@ ul {
 .cta {
   width: auto;
   margin: 24px 0 !important;
-  padding-left: 24px;
+  //padding-left: 24px;
   border-radius: 4px;
   border-left: 1px solid var(--app-sidebar-border-color);
-  margin-bottom: 40px;
+  //margin-bottom: 40px;
```
Comment (adamrrudolf) [inline css/main.scss:1704]: remove the code instead of commenting

File: css/main.scss [hunk 2]
```diff
@@ -1699,10 +1698,10 @@ ul {
 .cta {
   width: auto;
   margin: 24px 0 !important;
-  padding-left: 24px;
+  //padding-left: 24px;
```
Comment (adamrrudolf) [inline css/main.scss:1701]: remove

File: css/main.scss [hunk 3]
```diff
@@ -1796,9 +1795,8 @@ ul {
   }
 }
 fieldset {
-  width: 80%;
-  margin: 0 auto;
   border: none;
+  padding: 0rem;
```
Comment (adamrrudolf) [inline css/main.scss:1799]: is the removal of this 80% fieldset will keep the registration forms intact? Can you check or make this class specific to API explorer? cc @hubert-deriv

File: css/main.scss [hunk 4]
```diff
@@ -1857,11 +1855,11 @@ fieldset {
   }
 }
 textarea {
-  width: calc(100% - 16px);
+  width: 100%;
```
Comment (adamrrudolf) [inline css/main.scss:1858]: came about this css. It is unclear that "textarea" is only affecting API explorer. Can you change to class name specific to API explorer? cc @hubert-deriv

File: css/main.scss [hunk 5]
```diff
@@ -2051,7 +2049,7 @@ button {
   flex-direction: column;
 }
 .doc-content {
-  padding: 105px 0 80px;
+  padding: 105px 120px 80px;
```
Comment (adamrrudolf) [inline css/main.scss:2052]: 120px was there before, how come it is 0 again? Did you merged with latest changes?

**Review by adamrrudolf — CHANGES_REQUESTED** [review]

File: css/main.scss (hunk 3 above)
Comment (hubert-deriv) [reply css/main.scss:1799]: > is the removal of this 80% fieldset will keep the registration forms intact? Can you check or make this class specific to API explorer? cc @hubert-deriv

You make a good point here. I think this will break the form fields inside the app registration and maybe other places. I suggest reverting this CSS back to what it was and make a new separate specified class where you want to apply these changes.

File: css/main.scss [hunk 6]
```diff
@@ -2673,18 +2670,20 @@ pre {
...
+      
 
       fieldset {
         display: flex;
         flex-wrap: nowrap;
         align-items: center;
-        margin-right: 24px;
-        margin: 0 24px 0 0;
         justify-content: flex-start;
-        min-width: calc(60% + 10px);
```
Comment (hubert-deriv) [inline css/main.scss:2687]: I think here aswell, fieldset is a global selector, which means that you will change all the fieldset elements if you change something here. I think it's also better if you could revert these changes and create a new CSS selector that specifies the element you want to change and apply your CSS on there.

**Review by hubert-deriv — CHANGES_REQUESTED** [review]

**Review by hubert-deriv — APPROVED** [review]

**Review by adam-deriv — APPROVED** [review]
